### DISCUSSION TITLE
Does this support vue3?

### DISCUSSION THREAD

**Discussion by r0kk** [nhn/tui.chart#732] [Q&A]
I am new to the vue, and my question is if this supports vue3? There is not explicitly stated anywhere that it does.

If it does support, how can I make it work? I used your example but it breaks.

Thank you for the answer.

Kind Regard,
**consol_error_log_1:**
![consol_error_log_1](https://user-images.githubusercontent.com/36562019/158170299-a909995c-0e5a-47ef-8f48-3734773da9aa.png)
**consol_error_log_2(continued):**
![consol_error_log_2](https://user-images.githubusercontent.com/36562019/158170304-4a06cdbb-34d6-4f29-9d5b-ce582b55030e.png)
**yarn installation:**
![yarn_installation](https://user-images.githubusercontent.com/36562019/158170308-88188680-8bef-47dd-a1df-66e5a3a74e23.png)
**used example:**
![your_example](https://user-images.githubusercontent.com/36562019/158170310-b5be6458-73a2-4e19-b493-5610a405c00d.png)

Rok

**Comment by jwlee1108** [comment]
Sorry, our wrapper doesn't support vue 3 yet.
